docs/en-US/component/select.md

```diff
@@ -119,104 +119,110 @@ select/value-key
+| no-data-text                        | displayed text when there is no options, you can also use slot `empty`, default is 'No data'                                | ^[string]                                                                                                                                                                   | —                |
+| popper-class                        | custom class name for Select's dropdown                                                                                     | ^[string]                                                                                                                                                                   | ''               |
+| reserve-keyword                     | when `multiple` and `filter` is true, whether to reserve current keyword after selecting an option                          | ^[boolean]                                                                                                                                                                  | true             |
+| default-first-option                | select first matching option on enter key. Use with `filterable` or `remote`                                                | ^[boolean]                                                                                                                                                                  | false            |
+| teleported                          | whether select dropdown is teleported to the body                                                                           | ^[boolean]                                                                                                                                                                  | true             |
+| persistent                          | when select dropdown is inactive and `persistent` is `false`, select dropdown will be destroyed                             | ^[boolean]                                                                                                                                                                  | true             |
+| automatic-dropdown                  | for non-filterable Select, this prop decides if the option menu pops up when the input is focused                           | ^[boolean]                                                                                                                                                                  | false            |
+| clear-icon                          | Custom clear icon component                                                                                                 | ^[string] / ^[object]`Component`                                                                                                                                            | CircleClose      |
```

Comment:
`Custom clear icon component`  Perhaps the first letter is not capitalized, consistent with other descriptions.  And the comments in the code below are the same.

Reply:
updated